packages/components/src/Menu/Menu.tsx

```diff
@@ -86,13 +128,30 @@ export interface SectionProps {
   actions: ActionProps[];
 }
 
+export function useIsMobileDevice(): boolean {
+  if (typeof window === "undefined") {
+    return false;
+  }
+
+  // Unfortunately there's no way to tell RAC to use a different breakpoint
+  // so we're using the same one as the one they use
+  // To properly reference their value we need to also pull in react-spectrum
+  return window.screen.width <= REACT_ARIA_MOBILE_BREAKPOINT;
```

Comment:
don't love this but it's coming from internals that we simply cannot access or override.

https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/menu/src/MenuTrigger.tsx#L90-L95

Reply:
lol I don't wanna talk about it